README.md

```diff
@@ -148,11 +150,25 @@ Thanks for your interest in contributing!
 
 Our goal is to make contributing to this project easy for people. While we've made certain architectural decisions here to hopefully achieve that goal, it's a work in progress and feedback is appreciated.
 
-### Development environment
+## Development environment
+
+We recommend using Gitpod, a full development environment in the cloud.
```

Comment:
I don't think this language is agreed upon. "You may use Gitpod" is likely better.

Reply:
```suggestion
You may use Gitpod, a full development environment in the cloud.
```